draft-ietf-taps-interface.md

```diff
@@ -1124,6 +1125,31 @@ not all ICMP errors will necessarily be delivered, so applications cannot rely
 on receiving them.
 
 
+### Initiating side is not the first to write {#active-read-before-send}
+
+Name:
+: activeReadBeforeSend
+
+Type:
+: Preference
+
+Default:
+: Ignore
+
+The most common client-server communication pattern involves the
+client actively opening a connection, then sending data to the server. The
+server listens (passive open), reads, and then answers. This property
+specifies whether an application wants to diverge from this pattern -- either
+by actively opening with Initiate(), immediately followed by reading, or passively opening with Listen(),
+immediately followed by writing. This property is irrelevant for connection
+establishment using Rendezvous().
```

Comment:
"irrelevant" isn't right here - maybe "is ignored"?

Reply:
ACK, fixed with a commit to the master